packages/flutter/lib/src/widgets/image_filter.dart

```diff
@@ -46,32 +65,75 @@ class ImageFiltered extends SingleChildRenderObjectWidget {
   }
 }
 
-class _ImageFilterRenderObject extends RenderProxyBox {
-  _ImageFilterRenderObject(this._imageFilter);
+/// Applies an [ImageFilter] to its child.
+class RenderImageFiltered extends RenderProxyBox {
```

Comment:
if we're making this public it might make sense to move it to the rendering layer too, so people can use it without importing widgets if they're not using the widgets layer

Reply:
Other ROs like RenderTransform and RenderShaderMask are in proxy_box.dart - is that a good location?

Reply:
Moved to proxy_box.dart in the latest commit.